app/views/themes/cultural_show/hyrax/base/show.html.erb

```diff
@@ -3,37 +3,37 @@
 <%= render 'shared/citations' %>
 <%= render './shared/additional_citations' %>
 <div class="row">
-  <div itemscope itemtype="http://schema.org/CreativeWork" class="col-xs-12">
-    <div class="panel panel-default">
-      <div class="panel-heading">
+  <div itemscope itemtype="http://schema.org/CreativeWork" class="col-12">
+    <div class="card">
+      <div class="card-header">
         <%= render 'work_title', presenter: @presenter %>
       </div>
-      <div class="panel-body">
+      <div class="card-body">
         <div class="row">
           <%= render 'workflow_actions_widget', presenter: @presenter %>
-          <% if @presenter.universal_viewer? %>
-            <div class="col-sm-12 centered-media">
+          <% if @presenter.iiif_viewer? %>
+            <div class="col-12 centered-media">
               <%= render 'representative_media', presenter: @presenter, viewer: true %>
             </div>
           <% end %>
-          <div class="col-sm-12">
+          <div class="col-12">
             <%= render "show_actions", presenter: @presenter %>
           </div>
-          <div class="centered-media <%= !@presenter.universal_viewer? ? 'col-sm-6 text-center' : '' %>">
-            <%= render 'representative_media', presenter: @presenter, viewer: false unless @presenter.universal_viewer? %>
+          <div class="centered-media <%= !@presenter.iiif_viewer? ? 'col-md-6 text-center' : '' %>">
```

Comment:
Reading Note: Having just reconciled this for contribute back, this will be another wonderful collision.